static/app/icons/iconOption.tsx

```diff
@@ -0,0 +1,16 @@
+import {forwardRef} from 'react';
+
+import {SvgIcon, SVGIconProps} from './svgIcon';
+
+const IconOption = forwardRef<SVGSVGElement, SVGIconProps>((props, ref) => {
```

Comment:
Very small nitpick and more of a question for myself than anything else, but should this use the function declaration or is this page out of date? https://develop.sentry.dev/frontend/#defining-react-components

Reply:
Yes definitely! If this is how we've been doing all the other icons that's fine with me - just wanted to point it out if it was an oversight.